scripts/valhalla_build_extract

```diff
@@ -53,6 +53,38 @@ class TileHeader(ctypes.Structure):
     ]
 
 
+class TileResolver:
```

Comment:
a wrapper so we don't have to care about whether we're pulling tiles from a dir or tar